docs/redoc-http4s/src/main/scala/sttp/tapir/redoc/http4s/RedocHttp4s.scala

```diff
@@ -14,14 +14,22 @@ import scala.io.Source
   * @param yaml        The yaml with the OpenAPI documentation.
   * @param yamlName    The name of the file, through which the yaml documentation will be served. Defaults to `docs.yaml`.
   * @param redocVersion The semver version of Redoc to use. Defaults to `2.0.0-rc.23`.
+  * @param contextPath  The base path of the documentation
   */
-class RedocHttp4s(title: String, yaml: String, yamlName: String = "docs.yaml", redocVersion: String = "2.0.0-rc.23") {
+class SiloRedocHttp4s(
+                       title: String,
+                       yaml: String,
+                       yamlName: String = "docs.yaml",
+                       redocVersion: String = "2.0.0-rc.23",
+                       contextPath: String = "",
```

Comment:
this doesn't seem to be used anywhere?

Reply:
🤒  Sorry. Fixed.

Reply:
np :) I think the class name got changed as well? ;)

Reply:
more generally - maybe the `contextPath` could be a `List[String]`?

Reply:
It could. 
For the time being, we use a Htt4s Router on top to manage this need.